dotcom-rendering/fixtures/manual/calloutCampaignV2.ts

```diff
@@ -0,0 +1,258 @@
+import type { CalloutBlockElementV2 } from '../../src/types/content';
```

Comment:
This is the mock campaign for the story. It uses every type of callout field that we should expect.

Reply:
nice - good coverage